### PR TITLE
format: HEP brief search results fixes

### PR DESCRIPTION
**Pull request by nikpap**
Signed-off-by: Nick Papoutsis <admin@nils.gr>

### CODE REVIEW

File: inspirehep/modules/theme/static/scss/format/brief-results.scss [hunk 1]
```diff
@@ -179,11 +189,11 @@ hr {
 }
 
 .facet-container {
-  color: $grey;
+  color: $facet-gray;
 
   .facet-label {
-    background-color: $clouds;
-    color: $concrete;
+    background-color: #edf3f3;
```
Comment (jacquerie) [inline inspirehep/modules/theme/static/scss/format/brief-results.scss]: Maybe pull this one in a variable like you did with `facet-gray`?

**Comment by jacquerie** [comment]
LGTM, but needs a rebase, so maybe you can fix the nit and we can fast-merge this.

**Comment by nikpap** [comment]
@jacquerie 👍

**Comment by jacquerie** [comment]
:ship:

**Pull request merged**